ServerProxy/build.gradle

```diff
@@ -43,13 +43,13 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled true
-            proguardFiles 'proguard.cfg'
+            minifyEnabled false
```

Comment:
disabling this makes the APK larger then necessary.

I already tried fixing the proguard issues with bc0aa740ad98e70fd2a983d1c0120c8e7f063657

can you verify whether changing the next line is sufficient here?

Reply:
read the link. nevermind